conans/client/cache/cache.py

```diff
@@ -58,7 +59,7 @@ def __init__(self, base_folder, store_folder, output):
         super(ClientCache, self).__init__(self._store_folder)
         self.editable_packages = EditablePackages(self.conan_folder)
 
-    def package_layout(self, ref, short_paths=False):
+    def package_layout(self, ref, short_paths=None, *args, **kwargs):
```

Comment:
what is the meaning of `short_part=None` and difference from `short_path=True`? does it allow other values besides True/False?

Reply:
short_path=None => Use short-paths if they have been defined, but do not create new ones.
short_path=True=> Use short-paths, create new short-paths if not existing
short_path=False=>Never use short-paths